plugin.php

```diff
@@ -10,6 +10,7 @@
 Plugin URI: https://wordpress.org/plugins/seo-data-transporter/
 Description: SEO Data Transporter helps you transfer post/page specific SEO data, like custom doctitles, custom META descriptions and keywords, etc., from one platform (theme or plugin) to another.
 Version: 1.1.1
+Requires PHP: 8.1
```

Comment:
This is going to exclude some users, but PHP `8.0` is losing security support [in 22 days](https://www.php.net/supported-versions.php)